KUBERNETES.md

```diff
@@ -0,0 +1,231 @@
+## Running: Kubernetes
+
+This project uses [Lightbend's Platform Tooling](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/) to
+simplify deployment to [Kubernetes](https://kubernetes.io/). Follow the steps below to install Online Auction in your
+own local Kubernetes environment.
+
+## Setup
+
+You'll need to ensure that the following software is installed:
+
+* [Docker](https://www.docker.com/)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [Minikube](https://github.com/kubernetes/minikube) v0.25.0 or later (Verify with `minikube version`)
+* [Helm](https://github.com/kubernetes/helm)
+* [reactive-cli](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/cli-installation.html#install-the-cli) 0.9.0 or later (Verify with `rp version`)
+
+## Deploy
```

Comment:
I suggest changing this first heading from "Deploy" to "Choose deployment type" Then, maybe change the introduction to:
Once you've set up your environment, you can deploy online auction to Kubernetes. The steps are differ depending on whether you want to deploy in a development or production environment.

During development, it is simpler to deploy all services with a single sbt task. The <link>Developer Workflow section describes how to do this. It will build Docker images .... (include the description moved from the section into this intro).

The <link>Operations Workflow section describes the steps for deploying in a production environment. You will use sbt to build the images and the reactive-cli generates ... (pull up explanation from below). Because the production environment is more complex, additional steps are required as described below.

Reply:
incorporated